planner/functional_dependency/fd_graph.go

```diff
@@ -45,7 +45,37 @@ func (s *FDSet) closureOfStrict(colSet FastIntSet) FastIntSet {
 	return resultSet
 }
 
-// closureOf is to find strict equivalence closure of X with respect to F.
+// ClosureOfLax is used to find lax fd closure of X with respect to F.
+func (s *FDSet) closureOfLax(colSet FastIntSet) FastIntSet {
+	// Lax dependencies are not transitive (see figure 2.1 in the paper for
```

Comment:
We haven't mentioned this paper currently in our coments. We can add it at the header comment of the struct `FDSet`.